client/src/main/java/org/eclipse/hono/client/CommandClient.java

```diff
@@ -81,8 +81,8 @@
      *         <p>
      *         If the one-way command was accepted, the future will succeed.
      *         <p>
-     *         If the one-way command could not be processed by the protocol adapter, the future will fail with a {@link ServiceInvocationException} containing
-     *         the (error) status code. Status codes are defined at <a href="https://www.eclipse.org/hono/api/command-and-control-api">Command and Control API</a>.
+     *         The future will fail with a {@link ServiceInvocationException} if the one-way command could not be forwarded to the device.
+     *         The contained status code is defined at <a href="https://www.eclipse.org/hono/api/command-and-control-api">Command and Control API</a>.
```

Comment:
it is not, there is no such definition in the API for one-way commands because there is no response containing a status code. If there are any specific error codes then they will need to be defined here.

Reply:
Yes, you are right - the only error code is `503` (which is not generated by a response), so we do not need to specify it here IMHO.
Will remove this reference.